From acb527af5b14dd6cc527ccff6336b5b7115c16cc Mon Sep 17 00:00:00 2001
From: Davidson K <davidson.kumaresan@arm.com>
Date: Mon, 17 Apr 2023 16:16:37 +0530
Subject: [PATCH 8/9] virtio: add virtio block device

This will help in mounting a disk image as a virtio block device
---
 arch/arm/dts/total_compute.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/dts/total_compute.dts b/arch/arm/dts/total_compute.dts
index 9b2cbfb452..e08dd88bd9 100644
--- a/arch/arm/dts/total_compute.dts
+++ b/arch/arm/dts/total_compute.dts
@@ -56,4 +56,10 @@
 		method = "smc";
 		status = "okay";
 	};
+
+        virtio_block@1c130000 {
+                compatible = "virtio,mmio";
+                reg = <0x0 0x1c130000 0x0 0x200>;
+                interrupts = <0 170 4>;
+        };
 };
-- 
2.34.1

